### PR TITLE
Modernize a bit + update to get localization texts through module setup

### PR DESCRIPTION
**Pull request by osanoj**
- Update to get the needed localization texts through module setup instead of having the strings in the package. Modernize and update a lot of other things in the repo.
- Updated README
- Removed podspec
- Updated Fastfile to make version bumps based on tags
- Set minimum iOS 15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,21 +13,16 @@ Captured images are provided as an `Asset` which can retrieve image data asynchr
 
 ## Requirements
 
-* iOS 9.0
```
Comment (ninarg) [inline README.md:16]: wow 👻

File: Sources/Finjinon.swift [hunk 2]
```diff
@@ -0,0 +1,14 @@
+//
+//  Copyright (c) 2024 FINN.no AS. All rights reserved.
+//
+
+import Foundation
+
+public struct Finjinon {
+    static var configuration: FinjinonConfiguration!
+
+    /// Required setup to be able to use SearchQuest
```
Comment (ninarg) [inline Sources/Finjinon.swift]: SearchQuest?

Comment (osanoj) [reply Sources/Finjinon.swift]: Easy to guess where I copied those lines of code from 🙈😛

Comment (osanoj) [reply Sources/Finjinon.swift]: Fixed now 👍

**Review by ninarg — APPROVED** [review]

**Review by robinsalehjan — APPROVED** [review]
Great job!!

**Pull request merged**